src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -283,20 +283,44 @@
     "themes": [
         {
             "name": "light",
-            "window":{
+            "window":
+            {
                 "applicationTheme": "light"
+            },
+            "tab":
+            {
+                "background": "terminalBackground",
+                "unfocusedBackground": "#00000000"
+            },
+            "tabRow":
+            {
+                "unfocusedBackground": "#FFFFFFFF"

```

Comment:
this is very white

Reply:
aye, it is. There's not a lot of room to go lighter from `#e8e8e8`. I suppose we could wholesale copy Edge for light mode. That would be 
* `#cdcdcd` for focused
* `#e8e8e8` for unfocused

instead. With light mode we don't really have a "black tab on dark background" contrast issue we need to worry about. @cinnamon-msft thoughts?

Reply:
Are there light colors that are defined for the tab row control? I'd assume we'd want to align with XAML recommendations?

Reply:
... yes? The default WinUI colors are pretty terrible contrast-wise: `#f9f9f9` on `#f3f3f3`
![image](https://user-images.githubusercontent.com/18356694/185379047-1048aa8a-1234-4f62-a197-daef0798496d.png)

Which is part of the reason we had to change the colors in the first place 😐

Reply:
if Edge has good enough contrast, I vote we do that.